Zend/tests/type_declarations/typed_class_constants_004.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+Typed class constants (inheritance; simple)
+--FILE--
+<?php
+class A {
+    public const int A = 1;
+}
+
+class B extends A {
+    public const string A = 'a';
+}
+?>
+--EXPECTF--
+Fatal error: Type of B::A must be int (as in class A) in %s on line %d
```

Comment:
IMO:

```suggestion
Fatal error: B::A must be of type int (as in class A) in %s on line %d
```
And the error message for typed properties should also be adapted (not necessarily by you) :)